### PR TITLE
Blog about Survey

### PR DESCRIPTION
**Pull request by tombentley**
Signed-off-by: Tom Bentley <tbentley@redhat.com>

### CODE REVIEW

File: _posts/2020-09-25-strimzi-survey-results.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+Nearly 40% of respondants were using Strimzi in production.
+Over 50% were using it in a staging or development environment and
+about 15% were using it locally for development. 
+These don't sum to 100% because people could choose more than one option.
+
+## Kubernetes distributions and versions
+
+Over 55% of respondants were running Strimzi on OpenShift.
+This is likely a reflection of Red Hat being the company which started Strimzi.
+In comparison 25% were using it on EKS and the same on GKE.
```
Comment (ppatierno) [inline _posts/2020-09-25-strimzi-survey-results.md:38]: isn't it better to specify from which company EKS and GKE come? so Amazon and Google?

File: _posts/2020-09-25-strimzi-survey-results.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+## Authentication and authorization
+
+Most people said they were using TLS client authn, followed by Oauth authn and then SCRAM-SHA.
+
+Perhaps surprisingly Keycloak was the most popular solution for authorization and
+OPA was as popular as Kafka's native ACLs.
+
+This is another question where a answers from future survey will be interesting, given OPA's flexibility and typical enterprise authorization needs.
+
+## Docs
```
Comment (ppatierno) [inline _posts/2020-09-25-strimzi-survey-results.md:154]: Documentation?

File: _posts/2020-09-25-strimzi-survey-results.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+
+## Exposure
+
+* Using TLS for within-Kubernetes access is far-and-away the most common way of accessing the Kafka cluster.
+* For external access it was a fairly even split between routes, ingress or loadbalancer.
+* Relatively few people were using nodeport for out of cluster access, only 8%.
+
+## Authentication and authorization
+
+Most people said they were using TLS client authn, followed by Oauth authn and then SCRAM-SHA.
```
Comment (sjwoodman) [inline _posts/2020-09-25-strimzi-survey-results.md:147]: nit: expand authn to authentication

File: _posts/2020-09-25-strimzi-survey-results.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+## Images
+
+This was partly a question to tease out how many people have to deal with the project's functional-but-hardly-pretty `make` and `mvn` build system. 
+
+* Nearly 3/4 of people said they are using the images supplied by the project.
+* 10% are building custom images with modifications.
+
+In hidsight this was another ambiguous question because we don't really know why or whether people were counting having to build their own Kafka Connect images here or not.
+
+## Exposure
```
Comment (scholzj) [inline _posts/2020-09-25-strimzi-survey-results.md:139]: This sounds a bit weird to me ... maybe _Accessing Kafka_ would be better?

**Review by scholzj — APPROVED** [review]
One nit but LGTM

**Review by ppatierno — APPROVED** [review]

**Pull request merged**